webpack.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
```

Comment:
- [ ] You forgot to include the mode key and the value of development as part of the key-value pair that should be exported. This will cause an error when you start the application as shown in the screenshot below:

![image](https://user-images.githubusercontent.com/77106962/230218290-3c6a9dee-2f13-44db-85a0-e56f18a76b60.png)

#### Hint: You can add the key-value pair below before the `entry: './src/index.js',` pair

`mode: 'development',` 💻 🌵